tests/integration/files/file/base/file-pillarget-win.sls

```diff
@@ -0,0 +1,4 @@
+{% set file = salt['pillar.get']('monty', '') %}
+create_file:
+  file.managed:
+    - name: C:\\Windows\\Temp\\filepillar-{{ file }}
```

Comment:
Are all of these just so we can change to Windows paths? If so, would it be better just to use Jinja templating? I don't like the idea of maintaining a bunch of platform-specific SLS files if we don't need to.